src/bindings/js/node/tests/basic.test.js

```diff
@@ -0,0 +1,96 @@
+const ov = require('../build/Release/ov_node_addon.node');
+const path = require('path');
+
+function getModelPath(is_fp16=false){
+    basePath = "../../python/tests/"
+    if (is_fp16){
+        test_xml = path.join(basePath, "test_utils", "utils", "test_model_fp16.xml")
+        test_bin = path.join(basePath, "test_utils", "utils", "test_model_fp16.bin")
+    } else {
+        test_xml = path.join(basePath, "test_utils", "utils", "test_model_fp32.xml")
+        test_bin = path.join(basePath, "test_utils", "utils", "test_model_fp32.bin")
+    }
+    return (test_xml, test_bin)
+}
+
+
+
+var test_xml, test_bin = getModelPath();
+
+
+describe('Output class', () => {
+    
+    const core = new ov.Core();
+    const model = core.read_model(test_xml);
+    const compiledModel = core.compile_model(model, 'CPU');
+
+    it.each([
+        [model],
+        [compiledModel]
+    ])('Mutual tests', (obj) => {
+        expect(obj.output() && typeof obj.output() === 'object').toBe(true);
+        expect(obj.outputs.length).toEqual(1);  
+        //test for a obj with one output
+        expect(obj.output().toString()).toEqual("fc_out");
+        expect(obj.output(0).toString()).toEqual("fc_out");
+        expect(obj.output("fc_out").toString()).toEqual("fc_out");
+        expect(obj.output(0).shape).toEqual([1, 10]);
+        expect(obj.output(0).getShape().getData()).toEqual([1, 10]);
+        expect(obj.output().getAnyName()).toEqual("fc_out");
+    });
+
+    test('Ouput<ov::Node>.setNames() method', () => {
+        model.output().setNames(["bTestName", "cTestName"]);
```

Comment:
And single quotes instead on double

```suggestion
        model.output().setNames(['bTestName', 'cTestName']);
```